### PR TITLE
Bibata cursors update

### PR DESCRIPTION
**Pull request by AdsonCicilioti**
###### Motivation for this change
- **bibata-cursors**, **bibata-extra-cursors** and **bibata-cursors-translucent**: are very out of date. Upstream has updated them with noticeable visually perceived improvements.
- **clickgen** is a python-module and binary required to build Bibata Cursor Themes, created by the same upstream.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix
Comment (thiagokokada) [inline maintainers/maintainer-list.nix:285]: Looks fine.

```
$ curl -Ssq https://api.github.com/users/AdsonCicilioti | jq .id                                                     nix-shell
6278398
```

File: pkgs/data/icons/bibata-cursors/default.nix
Comment (thiagokokada) [inline pkgs/data/icons/bibata-cursors/default.nix:14]: Seems like the user changed its nickname, because https://github.com/KaizIqbal/Bibata_Cursor points to https://github.com/ful1e5/Bibata_Cursor.

**Review by thiagokokada — COMMENTED** [review]
Code wise looks fine.

File: pkgs/development/python-modules/clickgen/default.nix
Comment (WolfangAukang) [inline pkgs/development/python-modules/clickgen/default.nix:17]:
```suggestion
    repo = pname;
```

Minor nitpick, but repo has the same name as package.

File: pkgs/development/python-modules/clickgen/default.nix
Comment (WolfangAukang) [inline pkgs/development/python-modules/clickgen/default.nix:32]: What does bitmap error mean?

File: pkgs/development/python-modules/clickgen/default.nix
Comment (thiagokokada) [reply pkgs/development/python-modules/clickgen/default.nix:17]: If @AdsonCicilioti wants to do this feel free, however I don't care too much about this change (as you said, this is a nitpick).

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 149496` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>bibata-cursors</li>
    <li>bibata-cursors-translucent</li>
    <li>bibata-extra-cursors</li>
    <li>clickgen (python39Packages.clickgen)</li>
    <li>python38Packages.clickgen</li>
  </ul>
</details>

**Review by thiagokokada — APPROVED** [review]
Didn't test the themes itself, but it builds and the resulting files looks fine.

File: pkgs/development/python-modules/clickgen/default.nix
Comment (AdsonCicilioti) [reply pkgs/development/python-modules/clickgen/default.nix:17]: > 

I think this shouldn't be dynamic, despite being a simple name, upstream can change its default to capitalized for example. e Believed that simple name nixpkgs would always be lowercase. And if there's no pattern on both sides then I don't see much of it.

File: pkgs/development/python-modules/clickgen/default.nix
Comment (thiagokokada) [inline pkgs/development/python-modules/clickgen/default.nix:34]: With this patch the test seems to work fine:

```
diff --git a/pkgs/development/python-modules/clickgen/default.nix b/pkgs/development/python-modules/clickgen/default.nix
index 141fd4569b5..a85cee778b0 100644
--- a/pkgs/development/python-modules/clickgen/default.nix
+++ b/pkgs/development/python-modules/clickgen/default.nix
@@ -6,6 +6,7 @@
 , libXcursor
 , libpng
 , python
+, pytestCheckHook
 }:

 buildPythonPackage rec {
@@ -29,9 +30,7 @@ buildPythonPackage rec {
     install -m644 clickgen/xcursorgen.so $out/${python.sitePackages}/clickgen/xcursorgen.so
   '';

-  # Tests fail with ${bitmap_error}.
-  # Application works normally if tests are disabled
-  doCheck = false;
+  checkInputs = [ pytestCheckHook ];

   meta = with lib; {
     homepage = "https://github.com/ful1e5/clickgen";
```

Comment (AdsonCicilioti) [reply pkgs/development/python-modules/clickgen/default.nix:34]: Ok! That's work fine

File: pkgs/development/python-modules/clickgen/default.nix
Comment (AdsonCicilioti) [reply pkgs/development/python-modules/clickgen/default.nix:32]: The test don't found the bitmap directory. But i'll fix it now. Thx

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 149496` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>bibata-cursors</li>
    <li>bibata-cursors-translucent</li>
    <li>bibata-extra-cursors</li>
    <li>clickgen (python39Packages.clickgen)</li>
    <li>python38Packages.clickgen</li>
  </ul>
</details>

**Review by thiagokokada — APPROVED** [review]
Wait until the eval checks and I will merge.

**Comment by thiagokokada** [comment]
Thanks!

**Comment by wkral** [comment]
Ran into an issue with this update in that the name of the themes have changed.  Leaving this comment for other folks who run into this issue.

I was using `Bibata_Ice` in the various places in my configuration which has changed. There are now `Bibata-Original-Ice` and `Bibata-Modern-Ice`. The other varients seem to follow a similar pattern.

**Pull request merged**
